trendmicro_deepsecurity/plugin.spec.yaml

```
@@ -33,11 +33,19 @@ connection:
     type: string
     default: https://app.deepsecurity.trendmicro.com
     required: true
+    example: https://192.168.1.100:4119
```

Is 4119 the default port on-premise Deep Security uses? If so, then let use a test IP address (we're standardizing here, so bear with me). RFC5737

```suggestion
    example: https://198.51.100.100:4119
```

---

Oh cool I didn't know there is a reserved network for documentation. 
Yes that's the default port of the Deep Security Manager.